urumanifest/commit.py

```diff
@@ -109,7 +124,12 @@ def on_compress(asset: manifest.ManifestEntry, future: concurrent.futures.Future
                       for i in compressed_assets)
         for server_path, staged_asset, source_asset, cached_asset in asset_iter:
             assert server_path.parent.name
-            asset_output_path = output_path.joinpath(server_path).with_suffix(f"{server_path.suffix}.gz")
+
+            if source_asset.source_path.is_dir():
+                staged_asset.flags |= ManifestFlags.bundle
```

Comment:
It feels like we should be flagging this earlier in the process. This function is responsible only for compressing the assets.

Reply:
Not sure where would be better. It's weird to have this in a compression stage - but the bundle flag is also a direct result of how the compression stage decides to compress. I could see maybe driving it in reverse - something else marks it as a bundle and then uses the flag to do an alternative compression. I'm just not sure which function would be better for setting the flag.

Reply:
The correct place to set this is dependencies.py in `find_client_dependencies()` subfunction `track_manifest_dependency()`. Windows prerequisite installers are already being marked there based off their presence in the `prereq` sections. We probably want for the presence in the mac sections to work the same.